formatTest/typeCheckedTests/expected_output/comments.re

```diff
@@ -121,10 +131,13 @@ type typeParamPointWithComments 'a = {
 let name_equal x y => x == y;
 
 let equal i1 i2 =>
-  i1.contents === i2.contents && true /* most unlikely first */;
+  i1.contents === i2.contents &&
+    true /* most unlikely first */;
```

Comment:
I think this is a regression too. Because it is an end of line comment, and the "attachment location" totally includes an expression `x == y && z`, then the comment should be considered to be on that entire expression, while also preserving the fact that it's rendered on the end of the line.

When this happens, what we could do is maintain the end of line-ness, and then when rendering prevent *all* line breaks inside of the region that the end of line comment annotates. Even if we resize the text to be super skinny (`50` character width for example), the end of line comment will still appear on the exact same portion of the AST.

That would help us achieve the goal that we could reformat to a thin width, then resize to a wide width, and have all the comments in exactly the same place that they were originally.